Data/ByteString/Lazy/Char8.hs

```diff
@@ -568,6 +581,13 @@ findIndex :: (Char -> Bool) -> ByteString -> Maybe Int64
 findIndex f = L.findIndex (f . w2c)
 {-# INLINE findIndex #-}
 
+-- | The 'findIndexEnd' function takes a predicate and a 'ByteString' and
+-- returns the index of the last element in the ByteString
+-- satisfying the predicate.
+findIndexEnd :: (Char -> Bool) -> ByteString -> Maybe Int64
+findIndexEnd f = L.findIndexEnd (f . w2c)
+{-# INLINE findIndexEnd #-}
```

Comment:
Good catch! I did not realise that `findIndexEnd` is missing. Please add it to `Data.ByteString.Char8` as well, and write a test.